server_test.go

```diff
@@ -213,8 +213,8 @@ func TestShouldPeerBootstrap(t *testing.T) {
 	t.Parallel()
 
 	testCases := []struct {
-		cfg            *Config
-		shouldBoostrap bool
+		cfg             *Config
+		shouldBootstrap bool
```

Comment:
If we are touching code then it's not a typo fix, you need either create a dedicated commit or remove this change